src/resources/swagger.yaml

```diff
@@ -19,6 +19,49 @@ paths:
               schema:
                 type: object
 
+  /submission/admin-operations/recalculate-stats/{programId}/{submittedDonorId}:
```

Comment:
this should not be under submission, donors belong to /clinical/..
you can use `PATCH /clinical/donor/{donorId}/core-stats`
why are you using the submitter donor ID ?

Reply:
no need to mention `admin` in the url, it's validated in the authorization, I'd say it's better to keep roles names out of the api urls because something/someone else may use them in future

Reply:
you can however keep the tag as admin operations in swagger

Reply:
Yea this was a late day decision, didn't really think on it much so I just made a new router... I agree it should be in clinical will update :+1: